include/nbl/video/utilities/IGPUObjectFromAssetConverter.h

```diff
@@ -1036,7 +1041,10 @@ auto IGPUObjectFromAssetConverter::create(const asset::ICPUImage** const _begin,
 
         if (needToCompMipsForThisImg(cpuimg))
         {
-            params.usage |= asset::IImage::EUF_TRANSFER_SRC_BIT;
+            params.usage |= asset::IImage::EUF_TRANSFER_SRC_BIT; // this is for blit
+            // I'm already adding usage flags for mip-mapping compute shader
+            params.usage |= asset::IImage::EUF_SAMPLED_BIT; // to read source mips
+            params.usage |= asset::IImage::EUF_STORAGE_BIT; // to write dst mips
```

Comment:
am I likely to fuckup the format promotion with this?

Reply:
ok answered my own question with some testing, doing this causes an SRGB image to be promoted to 16bit UNORM